src/app/page.tsx

```diff
@@ -1,6 +1,6 @@
 "use client";
 
-import LandingPage from "@components/LandingPage";
+import LandingPage from "@components/landingPage";
```

Comment:
Issue is here 👆🏻

Reply:
Fixed !

Reply:
it looks the same in both files, the commit here doesn't seem to change it https://github.com/OlivierJM/sparked-next/pull/15/commits/47e82bbef12ed8348768b1ce0d05acf5a691c9ed